nutkit/protocol/responses.py

```diff
@@ -203,7 +203,7 @@ def __eq__(self, _):
                 return True
 
         from tests.shared import get_driver_name
-        if get_driver_name() in ["java", "javascript", "go", "dotnet"]:
+        if get_driver_name() in ["java", "javascript", "go", "dotnet", "ruby"]:
```

Comment:
If you driver has the option to provide this info, It'd be the preferred way to send the data. If not, this is fine.